adaptive_memory_v4.0.py

```
@@ -0,0 +1,1769 @@
+                else:
+                    logger.debug(f"Background summarization: skipped (enabled: {self.valves.enable_summarization_task}, users: {len(self.seen_users)})")
+
+            except asyncio.CancelledError:
+                logger.info("Summarization task cancelled")
+                break
+            except Exception as e:
+                logger.error(f"Summarization task error: {e}")
+                logger.error(f"Traceback: {traceback.format_exc()}")
+                await asyncio.sleep(60)
+
+    async def _log_error_counters_loop(self):
```

Ruff (0.14.10)

Missing return type annotation for private function `_log_error_counters_loop`

Add return type annotation: `None`

(ANN202)